gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -570,6 +591,17 @@ def main(args):
         action="store_true",
     )
 
+    parser.add_argument(
+        "--low-coverage-filter",
```

Comment:
This is a little confusing with ` "--lower-cov-cutoff"`

I'm not really sure of something better, maybe something like --pipeline-low-coverage-filter